tests/dualtor/tor_failure/test_tor_component_failure.py

```diff
@@ -0,0 +1,59 @@
+import pytest
```

Comment:
Should we save these files for a future PR? Not sure it makes sense to include them at the moment.

Reply:
Good suggestion. Removed them for later.